### DISCUSSION TITLE
(New Connector Profile) AWS::ApiGateway::RestApi > AWS::Serverless::StateMachine

### DISCUSSION THREAD

**Discussion by Khalilbohs** [aws/serverless-application-model#3609] [Ideas]
Currently, I'd have to make my own role and connect this within my api gateway for the step function.
I'd like there to be a more elegant way to do this, by using a connector.
There is already such a connector profile for serverless functions, using lambda functions.
I find that to have the approximate same use cases as for this new step function connector profile.

**Comment by gracelu0** [comment]
Hi, thanks for the feature request! Moving this to discussions where we track feature requests. This will not be prioritized at the moment, however, if it receives enough upvotes 👍 we will re-prioritize it.

**Reply by Khalilbohs** [reply]
Thank you!
